src/job.ts

```diff
@@ -338,15 +339,15 @@ export class Job {
     }
 
     get interactive (): boolean {
-        return this.jobData["interactive"] || false;
+        return this.jobData["gclInteractive"] || false;
     }
 
     get injectSSHAgent (): boolean {
-        return this.jobData["injectSSHAgent"] || false;
+        return this.jobData["gclInjectSSHAgent"] || false;
     }
 
     get description (): string {
-        return this.jobData["description"] ?? "";
+        return this.jobData["gclDescription"] ?? "";
```

Comment:
prefixed custom gitlab-ci-local decorator with gcl to prevent potential future name collision in json schema